templates/js-function/package.json

```
@@ -0,0 +1,11 @@
+{
+  "main": "index.js",
+  "name": "js-function",
```

```suggestion
  "name": "@scramjet/js-function",
```
Please apply this change `"name":` in each package.json file, thx 😉